plugins/fixed-edges/src/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/**
+ * @fileoverview Plugin overview.
+ */
+
+import Blockly from 'blockly/core';
+
+let fixedEdges = {
+  top: true,
+  left: true,
+};
+
+/**
+ * The manager for a workspace with fixed edges.
+ * @param {!Blockly.WorkspaceSvg} workspace The workspace to calculate metrics
+ *     for.
+ * @implements {Blockly.IMetricsManager}
+ * @constructor
+ */
+export class FixedEdgesMetricsManager extends Blockly.MetricsManager {
```

Comment:
The terminology of "fixed' may be confusing, so I'm not sure whether to switch to instead saying "constant".

Reply:
updated wording